src/appleseed/CMakeLists.txt

```diff
@@ -1238,6 +1238,8 @@ set (renderer_kernel_shading_sources
     renderer/kernel/shading/closures.h
     renderer/kernel/shading/directshadingcomponents.cpp
     renderer/kernel/shading/directshadingcomponents.h
+    renderer/kernel/shading/aovcomponents.cpp
+    renderer/kernel/shading/aovcomponents.h
```

Comment:
Would it make sense to move these two files to renderer/kernel/aovs/ ?
What do you think?